<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>INVOICE MANAGEMENT</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>1021</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Information Invoice</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>InvoiceID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CustomerName</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CreateAt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtInvoiceID">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCusName">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>StaffName</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtStaffName">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>40</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>120</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTotal">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>120</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateCreate">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>80</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAmount">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>80</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>1021</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Handle</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>801</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Search Invoice</string>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>33</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbxFilter">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Search By Code Order</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Search By Customer Name</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>33</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Key Word</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSearch">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearchDefault">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>27</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>801</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Filter Invoice</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Search By Price:     From</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Search By Date:     From</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPriceFrom">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>33</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPriceTo">
      <property name="geometry">
       <rect>
        <x>429</x>
        <y>30</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateFrom">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>72</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateTo">
      <property name="geometry">
       <rect>
        <x>429</x>
        <y>70</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearchByPrice">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>28</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearchByDate">
      <property name="geometry">
       <rect>
        <x>582</x>
        <y>67</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>501</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>List Invoice</string>
    </property>
    <widget class="QTableWidget" name="tblInvoice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>461</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReloadTblInvoice">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>15</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Reload</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>430</y>
      <width>511</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Detail Invoice</string>
    </property>
    <widget class="QTableWidget" name="tblDetailInvoice">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>481</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="txtUserName">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>918</x>
      <y>9</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
